sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -134,16 +134,21 @@ public static void setApiUrl(String apiUrl) {
      */
     public static <T> AppCenterFuture<Document<T>> create(String partition, String documentId, T document, Class<T> documentType) {
         AppCenterLog.debug(LOG_TAG, "Create started");
-        getInstance().instanceCreate(partition, documentId, document, documentType);
-        return null;
+        return getInstance().instanceCreateOrUpdate(partition, documentId, document, documentType);
     }
 
     /**
      * Replace a document.
      * The document instance (T) must be JSON serializable.
      */
-    public static <T> AppCenterFuture<Document<T>> replace(String partition, String documentId, T document) {
-        return null;
+    public static <T> AppCenterFuture<Document<T>> replace(String partition, String documentId, T document, Class<T> documentType) {
+        
+         /* 
+            In the current version we do not support E-tag optimistic concurrency logic and `replace` will call
```

Comment:
The right format for multiline comments is

```
  /*
   *  Line 1.   
   *  Line 2.
   */
```

There are many examples in other modules.

Reply:
Also end comments with `.`

Reply:
@guperrot @jwallra do you have any style-check configuration that I can use?